strongarm/common.py

```diff
@@ -126,22 +127,44 @@ def __expand(self):
 
         self.__next_url = data.get('next')
 
-        newData = [self.__content_cls(element) for element in data['results']]
-        self.__data += newData
+        new_data = [self.__content_cls(element) for element in data['results']]
+        self.__data += new_data
 
-        return newData
+        return new_data
 
     def __len__(self):
         return self.__len
 
     def __iter__(self):
+        # Track a set of objects based on unique attribute.
+        element_set = set()
         for element in self.__data:
```

Comment:
I don't love the duplicate code here, but I can't come up with a simple way to remove it. It seems that Python 3's `yield from` would be super useful here.